test-in-docker

```diff
@@ -15,13 +16,18 @@ term=screen-256color
 # ...see Modifiers in zshexpn(1) for details.
 
 # List of ZSH versions
-typeset -a versions
+typeset -aU versions
 versions=( docker/base-*/Dockerfile(N.on:h:t:s/base-//) )
+typeset -r versions
 
 # List of frameworks
-typeset -a frameworks
+typeset -aU frameworks
 frameworks=( docker/*/Dockerfile(N.on:h:t) )
-frameworks=${(@)frameworks:#base-*}
+for i in {$#frameworks..1}; do
+  # Remove all base entries
+  [[ "${frameworks[$i]}" = base-* ]] && frameworks[$i]=()
```

Comment:
Just a stylistic note: I would go here with double equals. That is more readable IMHO.

Reply:
Done in 3c27414e7e985536a8b286d09b2248e7dee8263e